pkg/sliceutils/diff.go

```diff
@@ -0,0 +1,43 @@
+package sliceutils
+
+// Diff returns, given two slices a and b sorted according to lessFunc, a slice of the elements occurring in a and b
+// only, respectively.
+func Diff[T any](a, b []T, lessFunc func(a, b T) bool) (aOnly, bOnly []T) {
```

Comment:
`Diff` vs `Difference` can be kind of confusing to differentiate without reading the doc

Reply:
Yeah, I renamed `Difference` to `Without`, which reads quite naturally IMHO:
```
tlsConf.NextProtos = sliceutils.Without(tlsConf.NextProtos, []string{"h2", alpn.PureGRPCALPNString})
```